data/data/libvirt/variables-libvirt.tf

```diff
@@ -50,3 +50,11 @@ variable "libvirt_bootstrap_memory" {
   default     = 2048
 }
 
+# Currently RHCOS maintain its default 16G size if that
+# changes we need to change it here also
+# https://github.com/coreos/coreos-assembler/pull/924
+variable "libvirt_master_size" {
+  type        = string
```

Comment:
maybe this can be an actual type like `number` instead of general string. https://www.terraform.io/docs/configuration/types.html#primitive-types

Reply:
@abhinavdahiya I did try the number but looks like from volume.string() return what we want then volume.size() so I went with string primitive.

Reply:
> @abhinavdahiya I did try the number but looks like from volume.string() return what we want then volume.size() so I went with string primitive.

@praveenkumar can you provide a little more context on which `volume.string() return what we want then volume.size()` are you talking about? 

and if you look at https://www.terraform.io/docs/configuration/types.html#conversion-of-primitive-types you will see that tf will do conversion for you but having a clearer API is preferred...

Reply:
I considered this during the initial review , but https://github.com/openshift/installer/blob/master/data/data/libvirt/variables-libvirt.tf is already quite inconsistent with string/number, it seems it has always been using `string` for int values except for the more recent addition there.